src/server/db_slice.cc

```diff
@@ -1145,15 +1144,18 @@ void DbSlice::FlushChangeToEarlierCallbacks(DbIndex db_ind, Iterator it, uint64_
   DVLOG(2) << "Running callbacks in dbid " << db_ind << " with bucket_version=" << bucket_version
            << ", upper_bound=" << upper_bound;
 
-  for (const auto& ccb : change_cb_) {
-    uint64_t cb_version = ccb.first;
+  const size_t limit = change_cb_.size();
```

Comment:
@BorysTheDev both of them are protected, `UnregisterOnChange` is synchronized so no issue here.